region/maridia/inner-yellow.json

```diff
@@ -2938,7 +2965,7 @@
                         "HiJump",
                         "canWalljump"
                       ]},
-                      "SpaceJump"
+                      "h_canFly"
```

Comment:
ScrewAttack would have to be separated out as an enemyKill method

Reply:
Good catch